http/src/main/resources/init-resources/rstudio-docker-compose-gce.yaml

```diff
@@ -28,6 +28,7 @@ services:
       - /var/custom_env_vars.env
     # See https://docs.docker.com/engine/reference/run/#user-memory-constraints
     mem_limit: ${MEM_LIMIT}   # hard limit on memory consumption by the container
+    memswap_limit: ${MEM_LIMIT}
```

Comment:
I was seeing this error when increasing the memory to ~60G:
```
Jul 20 00:31:07 saturn-222c020b-94b4-4f75-9cca-9203affc86c3 GCEMetadataScripts[451]: 2021/07/20 00:31:07 GCEMetadataScripts: startup-script: Error response from daemon: Cannot update container fedd86e891f95a351657188e3ef09c31818895a74b013c0674533f6be159646e: Memory limit should be smaller than already set memoryswap limit, update the memoryswap at the same time
```
Reading docs it feels reasonable to have mem limit == swap limit.